cpp/ql/src/semmle/code/cpp/dataflow/internal/DataFlowImpl.qll

```diff
@@ -258,8 +258,8 @@ private predicate additionalJumpStep(Node node1, Node node2, Configuration confi
 private predicate useFieldFlow(Configuration config) { config.fieldFlowBranchLimit() >= 1 }
 
 pragma[noinline]
-private ReturnPosition viableReturnPos(DataFlowCall call, ReturnKind kind) {
-  viableImpl(call) = result.getCallable() and
+private ReturnPosition viableReturnPos(DataFlowCall call, ReturnKindExt kind) {
+  viableCallable(call) = result.getCallable() and
```

Comment:
This means we no longer need the `viableImpl` predicate (right?). If so, I would like to have [viableImpl](https://github.com/Semmle/ql/blob/master/csharp/ql/src/semmle/code/csharp/dataflow/internal/DataFlowDispatch.qll#L110) renamed to `viableCallable`, and remove [this](https://github.com/Semmle/ql/blob/master/csharp/ql/src/semmle/code/csharp/dataflow/internal/DataFlowDispatch.qll#L185) alias. Perhaps a similar simplification is relevant for other languages.

Reply:
Yes.  Let's do that in a followup PR.

Reply:
OK.